src/rest/mormot.rest.server.pas

```diff
@@ -6509,6 +6522,21 @@ procedure TRestServer.ServiceMethodRegister(aMethodName: RawUtf8;
   end;
 end;
 
+procedure TRestServer.ServiceMethodUnregister(aMethodName: RawUTF8);
+var
+  ix: Integer;
+begin
+  aMethodName := TrimU(aMethodName);
+  if aMethodName='' then
+    raise EServiceException.CreateUTF8('%.ServiceMethodRegister('''')',[self]);
+  ix:=fPublishedMethods.FindHashed(aMethodName);
+  if ix<>-1 then begin

```

Comment:
hint: `if ix >=0 then` is safer and generates slightly better asm

Reply:
Done.